app/models/pieces.py

```diff
@@ -25,7 +26,21 @@ class Knight(Piece): ...
 class Bishop(Piece): ...
 
 
-class Rook(Piece): ...
+class Rook(Piece):
+    def validate_move(self, board: np.array, move: PieceMove):
+        super().validate_move(board, move)
+
+        # Check if the move is in the same column
```

Comment:
We could concatenate those two ifs and return True, but we could also split them for now (since this is the way we did for the other pieces as well) and refactor it later.